src/popup/container.js

```diff
@@ -442,6 +459,13 @@ class PopupContainer extends Component {
                     icon="help"
                     btnClass={styles.help}
                 />
+                <ButtonIcon
+                    href={`${constants.OVERVIEW_URL}#/`}
```

Comment:
This will need some way to make it so the notifs view is open by default. You could sync the `showInbox` state to the URL query state (via the overview feature's `enhancer` module) then just set it to open something like `${constants.OVERVIEW_URL}#/inbox` (just a suggestion; maybe better ways you're currently working on)